python/analysis/auto_generated/raster/qgsalignraster.sip.in

```diff
@@ -90,36 +90,9 @@ Gets raster value at the given coordinates (from the first band)
         RasterInfo( const QgsAlignRaster::RasterInfo &rh );
     };
 
-
-    enum ResampleAlg
-    {
-      RA_NearestNeighbour,
-      RA_Bilinear,
-      RA_Cubic,
-      RA_CubicSpline,
-      RA_Lanczos,
-      RA_Average,
-      RA_Mode,
-      RA_Max,
-      RA_Min,
-      RA_Median,
-      RA_Q1,
-      RA_Q3,
-    };
-
-    struct Item
```

Comment:
We'll also need to monkey patch this to QgsAlignRasterData.RasterItem

Reply:
Added